### ISSUE TITLE
An existing guidelines for Chinese historical document

### ISSUE THREAD

**Issue by knagasaki** [TEI-EAJ/Guidelines_for_Chinese#2]
I was kindly notified of a well-organized guidelines for TEI encoding of Chinese texts by Joey in DILA.
http://wiki.dila.edu.tw/pages/%E3%80%8A%E4%B8%AD%E5%9C%8B%E4%BD%9B%E5%AF%BA%E5%8F%B2%E5%BF%97%E3%80%8B%E6%A8%99%E8%A8%98%E4%BD%9C%E6%A5%AD

It would be a good start point to think about this matter.

So, I would like to ask to @duncdrum about possibilities of utilizing and sharing of it among your colleagues who are (or will be ) addressing in encoding Chinese texts. 

Hopefully Joey and his colleague, Ray will join here later.

**Comment by duncdrum** [comment]
@knagasaki i can share some ODDs once i have time to get back to this. But more generally, how are we supposed to utilize this github org and repo? So far it seems I m the only person not living in Tokyo here, and our discussions is invisible to the larger TEI community.  As I don't have push access to this repo, nor can I invite colleagues to this org, how should we go about extending this discussion?

**Comment by knagasaki** [comment]
> @knagasaki i can share some ODDs once i have time to get back to this. But more generally, how are we supposed to utilize this github org and repo? So far it seems I m the only person not living in Tokyo here, and our discussions is invisible to the larger TEI community. As I don't have push access to this repo, nor can I invite colleagues to this org, how should we go about extending this discussion?

A few of member of this repo are in Tokyo but the others in different areas in Japan. Moreover, I've invited two powerful TEIers in Taiwan and one in US. 
Anyway, I think the TEIers for Chinese historical documents should have a place to discuss to form a community and a repository which would be visible from the larger TEI community. I will invite more of them as far as I know and I will invite your colleague, if you would think it would be useful. And I will adjust permission of this repo if possible. (I'm sorry I'm not familiar with the permission system of Github so far.)

**Comment by knagasaki** [comment]
> @knagasaki i can share some ODDs once i have time to get back to this. But more generally, how are we supposed to utilize this github org and repo? So far it seems I m the only person not living in Tokyo here, and our discussions is invisible to the larger TEI community. As I don't have push access to this repo, nor can I invite colleagues to this org, how should we go about extending this discussion?

I've set your role in this repo as maintain. Please test it.

**Comment by duncdrum** [comment]
Yup got the notification, that works for this repo, and ~~should allow me to invite external collaborators~~.  So would you like there to be external collaborators, a team on the org for Chinese documents, how should interested people go about contributing?

**Comment by knagasaki** [comment]
Basically, I suppose you and your colleagues should encode Chinese historical documents according to the context of European DH such as DARIAH and CRALIN under ERIC. So, I think such work should be aligned more or less with the existing rules for Chinese documents. Then, as I hope at least you and the TEIers in Taiwan who I invited would talk about it, I invited you here. I may provide this repo continuously inviting external collaborators on encoding Chinese documents. This repo might be relatively easy and useful to discuss among TEIers for East Asian document. But if you would like to have another place to do it, I will ask my colleagues to go there. Anyway, as it is difficult to put many examples of encoding Chinese (and many other languages) in the general Guidelines, it is important to make a place to store examples in various languages and to put a prominent pointer in the ecosystem of the TEI.
